app/os_windows.go

```diff
@@ -86,6 +90,9 @@ var resources struct {
 	cursor syscall.Handle
 }
 
+// appID is the application unique identifier, set using `-appid` in Gogio.
+var appID = "GioWindow"
```

Comment:
What about `app.ID`? I agree that `app/internal/log.appID` should also be merged with this, but doesn't have to be in this PR.

Reply:
Sounds good to me.